src/templates/Navigator/index.js

```diff
@@ -1,12 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
 import Icon from 'atoms/Icon';
 import TextComponent from 'atoms/TextComponent';
 import StepProgress from 'molecules/StepProgress';
 import ContactCard from 'organisms/cards/ContactCard';
 import styles from './navigator.module.scss';
 
+
+function renderStepProgress() {
```

Comment:
I think having stepProgress in the navigator component is going to be difficult to use because we'll need to control it from the app side - we can probably just take it out here & have the end user implement it